ci-operator/jobs/openshift/cluster-image-registry-operator/openshift-cluster-image-registry-operator-master-presubmits.yaml

```diff
@@ -902,8 +907,8 @@ presubmits:
       ci-operator.openshift.io/prowgen-controlled: "true"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-cluster-image-registry-operator-master-e2e-vsphere-operator
-    optional: true
```

Comment:
Why drop `optional` here?

Reply:
We shouldn't merge PRs that break tests. I have a PR that makes these tests green, once it gets merged, we should keep them green.

If this PR will be merged before https://github.com/openshift/cluster-image-registry-operator/pull/458, we can override e2e-vsphere-operator results.

Reply:
Eventually I want to drop all `optional: true` from our jobs.

Reply:
@stevekuznetsov perhaps we need some clarification here.

If a job is marked `optional` but is triggered and then fails (ex: via a `/test` instruction), will that failing test block merge?

Reply:
If it's `optional`, its failures will not block merge.

You can use `always_run: false` if you want to run it only using a `/test` instruction. And `optional: false` if you don't want it to block merge.

Reply:
https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#requiring-jobs-for-auto-merge-through-tide